rulings/cards-rulings.yaml

```diff
@@ -271,7 +271,8 @@
     - "[vis] [VIS] The unlock effect takes place before the end of combat. [RTR 19970630]"
     - "[vis] [VIS] The vampires still unlock if an effect continues the combat or begins a new combat. [RTR 20020501] [LSJ 20030717]"
 100236|Bonding:
-    - "[DOM] The superior version cannot be used if you do not need the stealth at the time you play it. [TOM 19951109]"
+    - "[DOM] Cannot be used if you do not need the stealth at the time you play it. [TOM 19951109]"
+    - "[DOM] Cannot be used to increase the stealth of a non-bleed action. [LSJ 19980824] [RTR 19941109]"
```

Comment:
Replace your ruling on Power of One (now card text) by this one (still not in card text).